[2022-02-03 20:53:56,491] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-12-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:56,771] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-12-01T00:00:00+00:00 [queued]>
[2022-02-03 20:53:56,774] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:56,776] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-03 20:53:56,777] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-03 20:53:56,934] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-12-01 00:00:00+00:00
[2022-02-03 20:53:56,985] {standard_task_runner.py:52} INFO - Started process 6968 to run task
[2022-02-03 20:53:57,021] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_5', 'download_dataset_task', 'scheduled__2019-12-01T00:00:00+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_2.py', '--cfg-path', '/tmp/tmpdpl_0sgm', '--error-file', '/tmp/tmp5okxt_vt']
[2022-02-03 20:53:57,031] {standard_task_runner.py:77} INFO - Job 54: Subtask download_dataset_task
[2022-02-03 20:53:57,756] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_5.download_dataset_task scheduled__2019-12-01T00:00:00+00:00 [running]> on host 985445ce9491
[2022-02-03 20:53:58,439] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_5
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-12-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-12-01T00:00:00+00:00
[2022-02-03 20:53:58,454] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-03 20:53:58,459] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'pwd']
[2022-02-03 20:53:58,526] {subprocess.py:85} INFO - Output:
[2022-02-03 20:53:58,580] {subprocess.py:89} INFO - /tmp/***tmp_prn1k2r
[2022-02-03 20:53:58,586] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-03 20:53:59,336] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_5, task_id=download_dataset_task, execution_date=20191201T000000, start_date=20220203T205356, end_date=20220203T205359
[2022-02-03 20:53:59,545] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-03 20:53:59,900] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
